packages/fxa-content-server/app/scripts/templates/sign_in_token_code.mustache

```diff
@@ -1,16 +1,16 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="fxa-signin-code-header">{{#t}}Authorize this sign-in{{/t}}</h1>
+    <h1 id="fxa-signin-code-header">{{#unsafeTranslate}}Enter confirmation code<span class="description">for your Firefox account{{/unsafeTranslate}}</span></h1>
```

Comment:
This feels a bit... suspicious.
Should the `</span>` be inside the `{{/unsafeTranslate}}` block?
```suggestion
    <h1 id="fxa-signin-code-header">{{#unsafeTranslate}}Enter confirmation code<span class="description">for your Firefox account</span>{{/unsafeTranslate}}</h1>
```